xbmc/video/dialogs/GUIDialogVideoInfo.h

```diff
@@ -51,6 +51,8 @@ class CGUIDialogVideoInfo :
   static bool GetSetForMovie(const CFileItem* movieItem, std::shared_ptr<CFileItem>& selectedSet);
   static bool SetMovieSet(const CFileItem *movieItem, const CFileItem *selectedSet);
 
+  static bool SetVideoVersion(const std::shared_ptr<CFileItem> item);
```

Comment:
Please, var not by-val, but by ref (maybe only a typo?).

`const std::shared_ptr<CFileItem>` => `const std::shared_ptr<CFileItem>&`